SPECS/xo-lite.spec

```
@@ -0,0 +1,31 @@
+Summary: Xen Orchestra Lite
+Name:    xo-lite
+Version: 0.1.3
+Release: 1%{?dist}
+License: AGPL3-only
+URL:     https://github.com/vatesfr/xen-orchestra
+
+BuildArch: noarch
+
+Source0: https://github.com/vatesfr/xen-orchestra/releases/download/%{name}-v%{version}/%{name}-%{version}.tar.gz
+
+%description
+This package contains Xen Orchestra Lite, a lightweight version of the Xen
+Orchestra Appliance for single-host administration, running directly from your
+browser without having to deploy anything.
+
+%prep
+%autosetup -p1
+
+%install
+%{__install} -d -m 755 %{buildroot}/opt/xensource/www
+%{__cp} -a * %{buildroot}/opt/xensource/www
```

AFAIK macros with two underscores are just meant to be used internally by other macros. Their benefit/cost ratio in spec files is not very good. I haven't found any such macro in Fedora's Packaging Guidelines, for example.

I know they've been used a lot in the past, but I think they're out of fashing now.
```suggestion
install -d -m 755 %{buildroot}/opt/xensource/www
cp -a * %{buildroot}/opt/xensource/www
```

---

Fixed